mailet/crypto/src/main/java/org/apache/james/transport/matcher/IsSMIMESigned.java

```diff
@@ -45,8 +45,7 @@ public Collection<MailAddress> match(Mail mail) throws MessagingException {
         if (message == null) {
             return null;
         }
-        
-        
+
```

Comment:
Where are the (unit) tests for this class?

Reply:
From what I see there is no unit tests in general at all for mailets and matchers declared in the crypto module, everything seems tested on an integration test level?

Reply:
THis one is a simple mailet only checking content type and its logic is not well exercised by the integration tests.

I do believe unit test makes full sense here and are quick and easy to write.

Reply:
added